doc/release-notes/0.18/0.18.md

```diff
@@ -142,24 +153,10 @@ The v0.18 release contains the following known issues and limitations:
 <td valign="top">Manual builds on other platforms are possible by following our <a href="https://github.com/eclipse/openj9/blob/master/buildenv/Build_Instructions_V8.md">detailed build instructions</a>.</td>
 </tr>
 
-<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/897">#897</a></td>
-<td valign="top">Javadoc not building some OpenJ9 extensions.</td>
-<td valign="top">All platforms (OpenJDK 8 only)</td>
-<td valign="top">Some API documentation is missing in the <code>build/&lt;platform&gt;/docs</code> directory.</td>
-<td valign="top">None</td>
-</tr>
-
-<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/898">#898</a></td>
-<td valign="top">Javadoc building some unsupported OpenJ9 extensions.</td>
-<td valign="top">All platforms (OpenJDK 8 only)</td>
-<td valign="top">Some API documentation in <code>build/&lt;platform&gt;/docs</code> is not supported with OpenJ9.</td>
-<td valign="top">None</td>
-</tr>
-
 </tbody>
 </table>
 
 
 ## Other changes
 
-A full commit history for this release is available at [Eclipse OpenJ9 v0.18.0](https://github.com/eclipse/openj9/releases/tag/openj9-0.18.0).
+A full commit history for this release is available at [Eclipse OpenJ9 v0.18.0.0](https://github.com/eclipse/openj9/releases/tag/openj9-0.18.0).
```

Comment:
There is an extra `.0` here now.

Reply:
Urgh ... the hazards of global search and replace :-/ Fixed